service-node/backend/src/main/scala/org/enterprisedlt/fabric/service/node/Util.scala

```diff
@@ -213,6 +213,15 @@ object Util {
     //=========================================================================
     def mkDirs(path: String): Boolean = new File(path).mkdirs()
 
+
+    def ensureDirExists(path: String): Unit = {
+        val dir = new File(path)
+        if (!dir.exists()) {
```

Comment:
unnecessary check, call to this method is equal to call to `Util.mkDirs`

Reply:
ensureDirExists removed.